hack/generate.sh

```diff
@@ -26,14 +26,12 @@ ${K8S_CODE_GENERATOR}/generate-groups.sh client,informer,lister \
     "nfd:v1alpha1" --output-base=. \
     --go-header-file hack/boilerplate.go.txt
 
-rm -rf pkg/generated
-
-mv sigs.k8s.io/node-feature-discovery/pkg/generated pkg/
+rm -rf api/v1alpha1/{clientset,informers,listers}
```

Comment:
Shouldn't this be like

```suggestion
rm -rf api/generated
```

Reply:
Good catch!